application/cs/bootstrap.texy

```diff
@@ -174,8 +174,19 @@ $configurator->addStaticParameters([
 ]);
 ```
 
-Na parametr `projectId` se lze v konfiguraci odkázat obvyklým zápisem `%projectId%`. Třída Configurator automaticky přidává parametry `appDir`, `wwwDir`, `tempDir`, `vendorDir`, `debugMode` a `consoleMode`.
+Na parametr `projectId` se lze v konfiguraci odkázat obvyklým zápisem `%projectId%`.
 
+Defaultní statické parametry
+----------------------------
+
+Configurator přidává do konfigurace tyto užitečné statické parametry automaticky:
+
+- `$appDir` je absolutní souborová cesta do adresáře zdrojového kódu aplikace (např. `C://www/project/app`)
+- `$wwwDir` je absolutní souborová cesta do veřejně přístupného adresáře (např. `C://www/project/www`)
+- `$tempDir` je absolutní souborová cesta do adresáře pro dočasné soubory (např. `C://www/project/temp`)
+- `$vendorDir` je absolutní souborová cesta do adresáře knihoven (např. `C://www/project/vendor`)
+- `$debugMode` nám udává zda je aplikace v debugovacím režimu (true) nebo produkčním režimu (false)
+- `$consoleMode` je ??? (tohle nevím)
```

Comment:
consoleMode určuje, zda request přišel přes http nebo konzoli (`PHP_SAPI === 'cli')`. Použití je pro běžnou appku celkem vzácné. Např. nette/http jej používá pro určení, zda má startovat session. orisai/nette-console podle toho ví, zda má nahradit http request falešným.

Reply:
díky doplním